packages/expo-notifications/ios/EXNotifications/Notifications/Scheduling/EXNotificationSchedulerModule.m

```diff
@@ -69,15 +69,17 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
 }
 
 UM_EXPORT_METHOD_AS(cancelScheduledNotificationAsync,
-                     cancelNotification:(NSString *)identifier resolve:(UMPromiseResolveBlock)resolve rejecting:(UMPromiseRejectBlock)reject)
+                     cancelNotification:(NSString * _Nonnull)identifier resolve:(UMPromiseResolveBlock _Nonnull)resolve rejecting:(UMPromiseRejectBlock _Nonnull)reject)
```

Comment:
Unfortunately the `identifier` can be `null`.

In the past I remember problems in calling exported methods that specified nullability. Has this been fixed? Do these methods work ok even after adding the specifiers?